portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -372,6 +372,27 @@ definition {
 			value1 = "${validationErrorMessage}");
 	}
 
+	macro editValidationTargetArgument {
+		Pause(locator1 = "1000");
```

Comment:
@Jamersonze Is it possible to use `WaitForElementPresent(locator1 = "FormFields#VALIDATION_ARGUMENT");` instead, and avoid using Pause?